### PR TITLE
Add edit number answer

### PR DESCRIPTION
**Pull request by samiwel**
### What is the context of this PR?
Update the Publisher to handle conversion from Author Number to appropriate Runner type of Number.
Update the schemas in use by Publisher.
Updating the conversion process to meet the newest runner schema files.
Update Publisher to use units and currencies schema files during validation.

### How to review 
All tests and checks should pass.
Should be possible to convert an Author questionnaire with Number input type.

### CODE REVIEW

**Comment by samiwel** [comment]
[Add/edit number answer (1)](https://trello.com/c/uNCIkX74/68-add-edit-number-answer-1)

File: src/main.js [hunk 1]
```diff
@@ -24,8 +39,15 @@ app.get("/graphql/:questionnaireId(\\d+)", async (req, res, next) => {
...
   } catch (err) {
-    console.error(err);
-    next(err);
+    res.json(
+      merge(
+        {
+          result: "error",
+          message: err.message
+        },
+        { validation: err.validation }
```
Comment (WickyNilliams) [inline src/main.js]: Why merge?

```js
res.json({
  result: "error",
  message: err.message,
  validation: err.validation
});
```

File: src/main.js [hunk 2]
```diff
@@ -39,8 +61,15 @@ app.get("/publish/:questionnaireId(\\d+)", async (req, res, next) => {
...
+      )
+    );
```
Comment (WickyNilliams) [inline src/main.js]: Same here, no need to merge.

File: src/main.js (hunk 1 above)
Comment (samiwel) [reply src/main.js]: Good point, don't need merge anymore, will remove.

**Pull request merged**